bfh/fields.py

```diff
@@ -63,7 +64,10 @@ def __get__(self, instance, cls=None):
         return instance.__dict__.get(self.field_name)
 
     def __set__(self, instance, value):
-        instance.__dict__[self.field_name] = value
+        if value is None:
```

Comment:
I still think there might be an issue with the fact that you still never set the value to an instance of the callable... so every time you get, you get a new instance, so you can never edit the value if you have a default... I'm not 100% sure about this though, so I'll work on it.